src/archivematicaCommon/lib/bindpid.py

```diff
@@ -0,0 +1,459 @@
+#! /usr/bin/env python
+"""Bind PID. Command-line utility and module for requesting PID-binding against
+a Handle.net server.
+
+This module contains functionality for requesting PIDs from a Handle server
+and requesting that the created PID resolve to a specified URL. It makes
+Archivematica-specific assumptions and also allows for the requesting of
+*qualified* PID URLs (PURLs), i.e., qualified via URL query params like
+``?locatt=view:<QUALIFIER>``. It was designed against a particular API (that of
+IISH) but with the goal of generalizability in mind. For this reason, a fairly
+complex user configuration is required, i.e., user-supplied templates for
+resolve URLs and for PID-binding HTTP request bodies. See below.
+
+For usage run::
+
+    ./bindpid.py -h
+
+Example usage::
+
+    $ ./bindpid.py 58fe1b06-ee88-41ca-9bc9-320dcac6c858 file -c bindpid.cfg
+    Congratulations, you have successfully bound the handle (PID) "58fe1b06-ee88-41ca-9bc9-320dcac6c858". The following persistent URLs (PURLs) are now bound to the indicated URLs:
+    http://<RESOLVER_URL>/12345/58fe1b06-ee88-41ca-9bc9-320dcac6c858?locatt=view:access => https://my-domain.org/access/12345/58fe1b06-ee88-41ca-9bc9-320dcac6c858
+    http://<RESOLVER_URL>/12345/58fe1b06-ee88-41ca-9bc9-320dcac6c858?locatt=view:original => https://my-domain.org/original/12345/58fe1b06-ee88-41ca-9bc9-320dcac6c858
+    http://<RESOLVER_URL>/12345/58fe1b06-ee88-41ca-9bc9-320dcac6c858?locatt=view:preservation => https://my-domain.org/preservation/12345/58fe1b06-ee88-41ca-9bc9-320dcac6c858
+    http://<RESOLVER_URL>/12345/58fe1b06-ee88-41ca-9bc9-320dcac6c858 => https://my-domain.org/access/12345/58fe1b06-ee88-41ca-9bc9-320dcac6c858
+    Joels-MacBook-Pro-2:handlepid joeldunham
```

Comment:
It means I copy-pasted without editing :) (you mean just `Joels-MacBook-Pro-2:handlepid joeldunham`, right?)